app/form_models/unsubscribe_feedback_form.rb

```diff
@@ -0,0 +1,9 @@
+class UnsubscribeFeedbackForm
+  include ActiveModel::Model
+
+  attr_accessor :reason, :other_reason, :additional_info
+
+  validates :reason, inclusion: { in: UnsubscribeFeedback.reasons.keys }
```

Comment:
You should add the error messages for each attribute in `activerecord.yml`

Reply:
Yes I am aware of it, that's why I added the screenshot, so that the reviewer can easily spot the required text, and it is to add it to the translations